beep/dataset.py

```diff
@@ -339,6 +340,104 @@ def serialize(self):
         return self.dataset_dir
 
 
+def get_threshold_targets(dataset_diagnostic_properties,
+                          cycle_type_target="rpt_1C",
```

Comment:
Just to be consistent with the naming (with other modules), can we name this `cycle_type`?

Reply:
done!